staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -680,7 +673,15 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	if a.group.MetaGroupVersion != nil {
 		reqScope.MetaGroupVersion = *a.group.MetaGroupVersion
 	}
-	if a.group.OpenAPIModels != nil {
+
+	// Use TypeConverter's nil-ness as a proxy for whether SSA/OpenAPI is enabled
+	// This should be removed in the future and made unconditional
```

Comment:
When?

Reply:
https://github.com/kubernetes/kubernetes/pull/114998

Reply:
I'd mention that number in the comment personally.